### ISSUE TITLE
Optional World Session Instancing and Front Row Feature

### ISSUE THREAD

**Issue by MR-Alex42** [Frooxius/NeosPublic#350]
In order to allow an unlimited amount of users participating world session instancing would automatically start new world sessions when a maximum number of users is exceeded or the performance gets too bad. This should be an optional setting for worlds.

For bigger events it would be helpful to support a feature that AltspaceVR calls [front row](https://altvr.com/breakthrough-tech-gives-everyone-a-front-row-seat-at-events/).
This feature mirrors performers’ avatars (voice, movements, actions) to all world instance sessions
so all users can watch a performance even when being in different world sessions.

**Comment by Frooxius** [comment]
Yeah this could be interesting. There are two separate features though. One would be improvements for starting sessions. I want to combine it better with the world browsing, which could feature this. Question is who would be the host? If the user is looking to just join a session, they might inadvertently become a host.

I do plan to have something similar to the Front Row, but not exactly the same. It would essentially be a "streamed" world session, a one way stream that could easily be replicated and scaled to thousands and more of users, without having them join the session directly.

It would allow multiple users to still interact and use components to compute an "aggregate" of what other users are doing, to give feel of a large audience (e.g. people giving applause). 

This is a bigger architectural change though, so I'm not sure how soon will this come.

**Comment by MR-Alex42** [comment]
Thanks for the feeback. Thinking on combining the world session instancing with headless clients.
